.bazelci/presubmit.yml

```diff
@@ -1,16 +1,22 @@
 ---
 tasks:
   ubuntu1604:
+    shell_commands:
+    - pip3 install gitdb
```

Comment:
Could you try if `pip3 install -r third_party/requirements.txt` works? That way we don't have to update this each time something library breaks..

Reply:
Sounds good, let me try.

Reply:
It works!